azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/ArtifactHandlerImplV2.java

```diff
@@ -235,6 +253,40 @@ private String getResourceConfiguration() {
         return resources.stream().map(resource -> resource.toString()).collect(Collectors.joining(","));
     }
 
+    protected void deployExternalResources(DeployTarget deployTarget) throws AzureExecutionException {
+        final PublishingProfile publishingProfile = deployTarget.getPublishingProfile();
+        final String serverUrl = publishingProfile.ftpUrl().split("/", 2)[0];
+        try {
+            FTPClient ftpClient = FTPUtils.getFTPClient(serverUrl, publishingProfile.ftpUsername(), publishingProfile.ftpPassword());
+            List<Resource> externalResources = this.externalResources;
+            for (Resource resource : externalResources) {
```

Comment:
have you considered the different resources referring the same file? is duplicate file uploaded multiple times?

Reply:
Yes, but users could upload same file to different target folders.